README.md

```diff
@@ -1,9 +1,9 @@
-[Browser Automated Tests for Joomla! CMS(covering Users and Content features - GSoC1 16)](https://summerofcode.withgoogle.com/projects/#5724182314745856)
+[Browser Automated Tests for Joomla! CMS(covering Users and Content features - GSoC 16)](https://summerofcode.withgoogle.com/projects/#5724182314745856)
 ===
 
 Abstract
 ---
-System development nowadays more than ever starts to look for automated test methods. There are several main drivers for this trend -need for faster design‐develop‐test‐analysis cycle, push for higher quality, increasing complexity of systems and their integration and last but not least ever‐rising costs of manual testing. Software testing itself is very wide field with topics ranging from compliance frameworks.
+System development nowadays more than ever starts to look for automated test methods. There are several main drivers for this trend need for faster design‐develop‐test‐analysis cycle, push for higher quality, increasing complexity of systems and their integration and last but not least ever‐rising costs of manual testing. Software testing itself is very wide field with topics ranging from compliance frameworks.
```

Comment:
Make a bullet list of this:

> There are several main drivers for this trend
     - need for faster design‐develop‐test‐analysis cycle,
     - push for higher quality,
     - increasing complexity of systems and their integration and last but not least
     - ever‐rising costs of manual testing.

The last sentence is incomplete, like Yves already stated; also, I don't understand 'compliance frameworks' in this context.